src/scenes/Home/components/WeatherStateSearchBar.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react'
+import { FormControl } from 'react-bootstrap'
+
+export default function WeatherStateSearchBar (props) {
+  // eslint-disable-next-line react/prop-types
+  const { handleSearch } = props
+  const [text, setText] = useState('')
+  const onChange = (e) => {
+    setText(e.target.value)
+    handleSearch(e.target.value)
+    console.log('aaa', e.target.value)
```

Comment:
please remove console.log

Reply:
So sorry, i fogot remove it, now i removed